### PR TITLE
🔧 migrate oruga package to lib/ui

### PR DESCRIPTION
**Pull request by roiLeo**
- In preparation of #5611 
- `@oruga/oruga-ui` is now under `libs/ui` workspace
- fix `NeoIcon` size prop/class

### CODE REVIEW

File: libs/ui/src/components/NeoIcon/NeoIcon.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <o-icon
     :pack="pack || 'fas'"
     :icon="icon"
-    :size="size || 'small'"
+    :size="size || ''"
```
Comment (Jarsen136) [inline libs/ui/src/components/NeoIcon/NeoIcon.vue:5]:
```suggestion
    :size="size"
```

File: libs/ui/src/histoire.scss [hunk 2]
```diff
@@ -1,7 +1,9 @@
 /* // @import '@/styles/index.scss'; TODO: how to import this file? */
 @import 'bulma/bulma.sass';
-@import '@oruga-ui/oruga/dist/oruga.min.css';
-@import '@oruga-ui/oruga/dist/oruga-full-vars.min.css';
+// @import '@oruga-ui/oruga/dist/oruga.min.css';
+// @import '@oruga-ui/oruga/dist/oruga-full-vars.min.css';
+@import "~@oruga-ui/oruga/src/scss/oruga";
```
Comment (Jarsen136) [inline libs/ui/src/histoire.scss:5]:
```suggestion
@import "@oruga-ui/oruga/src/scss/oruga";
```

**Review by Jarsen136 — APPROVED** [review]
small stuff

otherwise lgtm

File: libs/ui/src/components/NeoIcon/NeoIcon.vue (hunk 1 above)
Comment (roiLeo) [reply libs/ui/src/components/NeoIcon/NeoIcon.vue:5]: default size isn't in `'small' | 'medium' | 'large'`

Comment (Jarsen136) [reply libs/ui/src/components/NeoIcon/NeoIcon.vue:5]: > default size isn't in 'small' | 'medium' | 'large'

If so, we should define the correct type for the size props in line 14.  Replacing 'small' with an empty string is not a good solution.

> so would it work if provide empty?
If it does not work, we should provide a proper default value for it.

File: libs/ui/src/histoire.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /* // @import '@/styles/index.scss'; TODO: how to import this file? */
 @import 'bulma/bulma.sass';
-@import '@oruga-ui/oruga/dist/oruga.min.css';
-@import '@oruga-ui/oruga/dist/oruga-full-vars.min.css';
+@import "@oruga-ui/oruga/src/scss/oruga";
```
Comment (roiLeo) [inline libs/ui/src/histoire.scss:3]:
```suggestion
@import '@oruga-ui/oruga/src/scss/oruga';
```

File: libs/ui/src/components/NeoIcon/NeoIcon.vue (hunk 1 above)
Comment (roiLeo) [inline libs/ui/src/components/NeoIcon/NeoIcon.vue:5]:
```suggestion
    :size="size || 'default'"
```
https://github.com/oruga-ui/oruga/blob/4915c4dbcb1a6c2cd39bc7660d41916b330bc0cd/packages/oruga/src/utils/icons.js#L46

**Review by Jarsen136 — APPROVED** [review]

**Pull request merged**